### PR TITLE
[RFR] Remove BulkActions in V3

### PR DESCRIPTION
**Pull request by Kunnu01**
- [x] Remove BulkActions

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -173,6 +173,7 @@ Components deprecated in 2.X have been removed in 3.x. This includes:
 * `ViewTitle` (use `Title` instead)
 * `RecordTitle` (use `TitleForRecord` instead)
 * `TitleDeprecated` (use `Title` instead)
+* `BulkActions` (use `BulkActionButtons` props instead)
```
Comment (Kmaschta) [inline UPGRADE.md]:
```suggestion
* `BulkActions` (use the [`bulkActionButtons` prop](https://marmelab.com/react-admin/List.html#bulk-action-buttons) instead)
```

**Comment by Kunnu01** [comment]
> Removing this component also implies updating the `List` component which used it

`List` uses a prop name `bulkActions`. Are you talking about that?

**Comment by djhi** [comment]
> List uses a prop name bulkActions. Are you talking about that?

Yes 

> If yes, ListToolbar also uses it. Should I remove it from both the files?

Yes :)

File: packages/ra-ui-materialui/src/list/ListToolbar.js [hunk 2]
```diff
@@ -52,7 +51,6 @@ const ListToolbar = ({
                 React.cloneElement(actions, {
                     ...rest,
                     className: styles.actions,
-                    bulkActions,
```
Comment (djhi) [inline packages/ra-ui-materialui/src/list/ListToolbar.js]: I think you'll have to update the default ListActions too

**Comment by fzaninotto** [comment]
Neat! I think the Upgrade guide should be a bit more explicit, by showing a code snippet with a before/after diff.

**Comment by djhi** [comment]
Do you mean we should remove all other deprecated components (LongTextInput for example)?

**Comment by Kunnu01** [comment]
Okay, I'll update it. Do you want it in a specific format or should I just simply add code snippets saying `before` and `after`?

**Comment by fzaninotto** [comment]
Check the rest of the UPGRADE file: we use snippets with the `diff` syntax. Try to do the same.

**Comment by Kunnu01** [comment]
Done. Please review. :)

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**